.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Libra Developers Website Build and Push
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out the Libra website repository under $GITHUB_WORKSPACE, so job can access it
+    - uses: actions/checkout@v2
+    # Installs node and yarn
+    - name: Use Node.js 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    # Install git
+    - name: Install git
+      run: |
+        apt --assume-yes update 
+        apt --assume-yes install git
+    # Deploy the website
+    - name: Deploying Libra Website to GitHub Pages
+      run: |
+        git config --global user.email "libra-doc-bot@users.noreply.github.com"
+        git config --global user.name "Libra Website Deployment Script"
+        echo "machine github.com login libra-doc-bot password $PUBLISH_WEBSITE_TOKEN" > ~/.netrc
```

Comment:
This looks good but before we can merge this we're going to need to move the `$PUBLISH_WEBSITE_TOKEN` from Circle to Github.